.github/workflows/ci-backend-hbase.yml

```diff
@@ -62,25 +62,13 @@ jobs:
         include:
           - module: hbase
             args: "-Dtest=\"**/diskstorage/hbase/*\""
-            name: hbase1-diskstorage
+            name: hbase-diskstorage
           - module: hbase
             args: "-Dtest=\"**/graphdb/hbase/*\""
-            name: hbase1-graphdb
+            name: hbase-graphdb
           - module: hbase
             args: "-Dtest=\"**/hadoop/*\""
-            name: hbase1-hadoop
-          - module: hbase
-            install-args: "-Dhbase.profile -Phbase2"
-            args: "-Dtest=\"**/diskstorage/hbase/*\""
-            name: hbase2-diskstorage
-          - module: hbase
-            install-args: "-Dhbase.profile -Phbase2"
-            args: "-Dtest=\"**/graphdb/hbase/*\""
-            name: hbase2-graphdb
-          - module: hbase
-            install-args: "-Dhbase.profile -Phbase2"
-            args: "-Dtest=\"**/hadoop/*\""
-            name: hbase2-hadoop
+            name: hbase-hadoop
```

Comment:
I'm not sure yet as I didn't read about HBase compatibility but if we decide to support HBase 3 after it becomes GA, will we have `hbase-hadoop` and `hbase3-hadoop`? If so I think it would make sense to have `hbase2-hadoop` and `hbase3-hadoop`.
Currently HBase 3 is in Alpha: https://github.com/apache/hbase/releases/tag/rel%2F3.0.0-alpha-1
That said, in case we can use HBase 3 client libraries to work with HBase to than it totally makes sense to have just one module `hbase-hadoop`.
I also didn't check if we can use hadoop3 libs to work with hadoop2. Right now JanusGraph supports Hadoop 2 because of TinkerPop but TinkerPop bumps Hadoop to 3 version in 3.6.0-SNAPSHOT version. Thus, I think JanusGraph will need to support both Hadoop 3 and Hadoop 2. In this case we just need to understand compatibility because we will have:
Hadoop 2
Hadoop 3
HBase 2
HBase 3

You can ignore this comment, it's just something we will need to think about later

Reply:
I think it is a good idea to mark the tests as hbase2-... .